libbeat/processors/add_process_metadata/config.go

```diff
@@ -101,6 +101,7 @@ func defaultConfig() config {
 		HostPath:              "/",
 		CgroupPrefixes:        []string{"/kubepods", "/docker"},
 		CgroupCacheExpireTime: cacheExpiration,
+		CgroupRegex:           "([0-9a-f]{64})(\\.scope)?$",
```

Comment:
```suggestion
		CgroupRegex:           "[-/]([0-9a-f]{64})(\\.scope)?$",
```

To further help limit false positives.

Reply:
Moved this into `gosigar_cid_provider.go` to avoid breaking change